javascript/opendcs-web-ui-react/build.gradle

```diff
@@ -0,0 +1,61 @@
+plugins {
+    id 'base'
+    id 'opendcs.deps-conventions'
+    id 'opendcs.sonar-conventions'
+    id 'opendcs.version-conventions'
+    id 'com.github.node-gradle.node' version '7.1.0'
+}
+
+
+
+configurations {
+    api
+}
+
+dependencies {
+    api(project(":api-client-typescript"))
+}
+
+
+
+def buildUi = tasks.register("buildUi", NpxTask) {
+    dependsOn genTask
```

Comment:
`dependsOn genTask` and `dependsOn api` are placeholders for now? after removing those I was able to run and see the app